manim/mobject/mobject.py

```diff
@@ -1295,3 +1306,19 @@ class Group(Mobject):
     def __init__(self, *mobjects, **kwargs):
         Mobject.__init__(self, **kwargs)
         self.add(*mobjects)
+
+
+class _AnimationBuilder:
+    def __init__(self, mobject):
+        self.mobject = mobject
+
+    def __getattr__(self, method_name):
+        from ..animation.transform import _MethodAnimation
```

Comment:
This is one of very few places where we have a local import; is there a circular import problem when adding it as a global one?

Reply:
Yes mobject.py is imported in animation.py